projects/video-client/src/lib/video-client.service.ts

```diff
@@ -417,7 +417,7 @@ export class VideoClientService {
                         this._vidyoClient.getConnectState()
                             .then(_status => {
                                 if (_status.webrtc) {
-                                    this._vidyoConnector.SelectLocalWindowShare(localWindowShare)
+                                    this._vidyoConnector.SelectLocalWindowShare({ localWindowShare:localWindowShare })
```

Comment:
Can be written like this { localWindowShare } in ES6
https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand